enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,1050 @@
+// conditions. Consumers interested in the set of updates that are
+// actually recommended for this cluster should use
+// availableUpdates. This list may be empty if no updates are
+// recommended, if the update service is unavailable, or if an empty
+// or invalid channel has been specified.
+// +optional
+conditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
+```
+
+The `availableUpdates` documentation will be adjusted to read:
+
+```go
+// availableUpdates contains updates recommended for this
+// cluster. Updates which appear in conditionalUpdates but not in
+// availableUpdates may expose this cluster to known issues. This list
+// may be empty if no updates are recommended, if the update service
+// is unavailable, or if an invalid channel has been specified.
+```
+
+The new ConditionalUpdate type will have the following schema:
+
+```go
+// ConditionalUpdate represents an update which is recommended to some
+// clusters on the version the current cluster is reconciling, but which
+// may not be recommended for the current cluster.
+// +k8s:deepcopy-gen=true
+type ConditionalUpdate struct {
+	// release is the target of the update.
+	// +required
+	release Release `json:"release"`
+
+	// risks represents the range of issues associated with
+	// updating to the target release. The cluster-version
+	// operator will evaluate all entries, and only recommend the
+	// update if there is at least one entry and all entries
+	// recommend the update.
+	// +optional
+	Risks []ConditionalUpdateRisk `json:"risks,omitempty"`
+
+	// conditions represents the observations of the conditional update's
+	// current status. Known types are:
+	// * Evaluating, for whether the cluster-version operator will attempt to evaluate any risks[].matchingRules.
+	// * Recommended, for whether the update is recommended for the current cluster.
+	// +patchMergeKey=type
+	// +patchStrategy=merge
+	// +listType=map
+	// +listMapKey=type
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+}
+```
+
+The new ConditionalUpdateRisk type will have the following schema:
+
+```go
+// ConditionalUpdateRisk represents a reason and cluster-state
+// for not recommending a conditional update.
+// +k8s:deepcopy-gen=true
+type ConditionalUpdateRisk struct {
+	// url contains information about this risk.
+	// +optional
+	URL URL `json:"url,omitempty"`
+
+	// reason is the CamelCase reason for not recommending a
+	// conditional update, in the event that matchingRules match the
+	// cluster state.
+	// +optional
+	Reason string `json:"reason,omitempty"`
+
+	// message provides additional information about the risk of
+	// updating, in the event that matchingRules match the cluster
+	// state. This is only to be consumed by humans. It may
+	// contain Line Feed characters (U+000A), which should be
+	// rendered as new lines.
+	// +optional
```

Comment:
I think you should require this

Reply:
Why be more strict than [my consumer][1]?  I agree from a policy perspective that we are unlikely to leave this empty. 
 Although I still don't understand what was going in with openshift/cincinnati-graph-data#918.  But then again, that was a tombstone, and not a blocked edge.

[1]: https://github.com/openshift/api/blob/67c28690af52a69e0b8fa565916fe1b9b7f52f10/config/v1/types_cluster_operator.go#L138